apis/experimentapi/experimentapi.yaml

```diff
@@ -0,0 +1,661 @@
+                      responseCode: OK
+                      result: null
+          '500':
+            description: Internal Server Error example for batch create
+            content:
+              application/json:
+                schema:
+                  $ref: '#/components/schemas/Experiment-500InternalServerError'
+                example:
+                  id: ekstep.analytics.experiment.submit
+                  ver: v1
+                  ts: '2020-11-23 15:15:18:331+0000'
+                  params:
+                    resmsgid: null
+                    msgid: null
+                    err: INTERNAL_ERROR
+                    status: SERVER_ERROR
+                    errmsg: 'Process failed,please try again later.'
+                  responseCode: SERVER_ERROR
+                  result: {}
+        deprecated: false
+  security:
+    - bearerAuth: []      
+    
+  components:
+    securitySchemes:
+      bearerAuth:
+        type: http
+        scheme: bearer
+        bearerFormat: JWT
+    schemas:
+      Params:
+        title: Params
+        required:
+          - resmsgid
+          - err
+          - status
+        type: object
+        properties:
+          resmsgid:
+            type: string
+            nullable: true
+          err:
+            type: string
+          status:
+            type: string
+      ErrorParams:
+        title: ErrorParams
+        required:
+          - resmsgid
+          - err
+          - status
+          - errorMsg
+        type: object
+        properties:
+          resmsgid:
+            type: string
+          err:
+            type: string
+          status:
+            type: string 
+          errorMsg:
+            type: object
+            properties:
+              status:
+                type: string
+      CreateExperiment:
+        title: CreateExperiment
+        required:
+          - request
+        type: object
+        properties:
+          request:
+            $ref: '#/components/schemas/Experiment-Request'
+        example:
+          request:
+            expId: U1234
+            name: USER_ORG
+            description: Experiment to get users with orgname as sunbird to explore page 
+            createdBy: User1
+            criteria: 
+              type: user
+              filters:
+                organisations.orgName: [sunbird]
+            data:
+              startDate: '2020-10-11'
+              endDate: '2020-10-20'
+              key: 'ASDSDFDK23SDFKJDK'
+              client: app
+              
+      Experiment-Request:
+        title: Experiment-Request
+        required:
+         - expId
+         - name
+         - createdBy
+         - description
+         - criteria
+         - data
+        type: object
+        properties:
+          expId:
+           type: string
+           description: Unique id for experiment
+          name:
+           type: string
+           description: name of the experiment
```

Comment:
Capitalize the first letter of the description and keep it consistent across all description entries.